src/OpenRiaServices.Tools/Test/Utilities/UnitTestTraceListener.cs

```diff
@@ -37,6 +37,7 @@ public static void Initialize(TestContext context, bool failOnDebugAssert)
                 Context = context;
                 FailOnDebugAssert = failOnDebugAssert;
                 Listener = new UnitTestTraceListener();
+#if NETFRAMEWORK
```

Comment:
Why only netframework?

Reply:
unsure, it might be that Debug.Listeners is not availible in net6 ?

We might want to at least look into what feature we might miss by removing code for net6